Bio/PDB/Atom.py

```diff
@@ -308,23 +308,23 @@ def set_serial_number(self, n):
         """Set serial number."""
         self.serial_number = n
 
-    def set_bfactor(self, bfactor):
+    def set_bfactor(self, bfactor: Optional[float]):
```

Comment:
We still support Python 3.9 so can't use the https://peps.python.org/pep-0604/ syntax yet. On the bright side, pyupgrade and other tools can update this for us.

https://github.com/asottile/pyupgrade?tab=readme-ov-file#pep-604-typing-rewrites